packages/app/lib/src/features/home/presentation/home/post.dart

```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: prefer_expression_function_bodies
```

Comment:
Why do you keep adding this?

Reply:
My vs code keeps flagging it and trying to reformat all of the code every time I save if I don't have it ignore it.